build/package.props

```diff
@@ -6,7 +6,7 @@
     <PackageProjectUrl>http://go.microsoft.com/fwlink/?LinkID=320972</PackageProjectUrl>
     <RepositoryUrl>https://github.com/Azure/azure-functions-host</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
-    <Copyright>� Microsoft Corporation. All rights reserved.</Copyright>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
```

Comment:
?

Reply:
Changed the encoding from ANSI to UTF-8. The copyright was showing up as:
![image](https://user-images.githubusercontent.com/1089915/52662949-61232a00-2eba-11e9-8f1d-699c2ba46ec3.png)

Now it's:
![image](https://user-images.githubusercontent.com/1089915/52662987-7009dc80-2eba-11e9-85a0-6c5a2bb692f9.png)

Reply:
(that was in NuGet Package Explorer).

Reply:
👍  I figured it was an encoding change (that's what I was asking you on Teams :) ), just wanted to confirm.